src/pages/developers/developer-toolkit-jak-zintegrować-tbtc-z-defi-app.pl.md

```diff
@@ -0,0 +1,39 @@
+---
+template: resource
+title: 'Developer Toolkit: Jak zintegrować tBTC z DeFi App'
+heading: 'Developer Toolkit: Jak zintegrować tBTC z DeFi App'
+description: >-
+  tBTC to łatwy sposób na dodanie Bitcoinów do Twojej dApp, dzięki czemu możesz
+  ją udostępnić posiadaczom BTC i zwiększyć wzrost liczby użytkowników. Poprzez
+  integrację TBTC, developerzy aplikacji DeFi na Ethereum umożliwiają
+  posiadaczom BTC zarabiać na odsetkach lub zaciągać pożyczki bez polegania na
+  scentralizowanych usługach.
+date: 2020-07-24T15:32:29.807Z
+tags:
+  - developers
+---
+tBTC to łatwy sposób na dodanie Bitcoinów do Twojej dApp, dzięki czemu możesz ją udostępnić posiadaczom BTC i zwiększyć wzrost liczby użytkowników. Poprzez integrację TBTC, developerzy aplikacji DeFi na Ethereum umożliwiają posiadaczom BTC zarabiać na odsetkach lub zaciągać pożyczki bez polegania na scentralizowanych usługach.
```

Comment:
*zarabiać* czy raczej *zarabianie*? Jeżeli umożliwiają to powiedziałbym że to drugie.

Reply:
Podobny komentarz odnośnie *zaciągać pożyczki* - powiedziałbym raczej *zaciąganie pożyczek*.